src/mixins/EditorMixin.js

```diff
@@ -58,29 +60,153 @@ export default {
 		}
 	},
 	computed: {
+		/**
+		 * Returns the events title or an empty string if the event is still loading
+		 *
+		 * @returns {string}
+		 */
+		title() {
+			if (!this.eventComponent) {
+				return ''
+			}
+
+			return this.calendarObjectInstance.title || ''
+		},
+		/**
+		 * Returns the location or null if the event is still loading
+		 *
+		 * @returns {string|null}
+		 */
+		location() {
+			if (!this.calendarObjectInstance) {
+				return null
+			}
+
+			return this.calendarObjectInstance.location
```

Comment:
Optional chaining would be nice in these case, I'm not sure if we have it through Babel.